akka-stream/src/main/scala/akka/stream/impl/FanIn.scala

```diff
@@ -292,10 +291,3 @@ private[akka] abstract class FanIn(val settings: ActorMaterializerSettings, val
 
 }
 
```

Comment:
cannot this whole file go? Is there something reused from here by stream-of-stream ops?